examples/cosmodc2/pipeline_redmagic_no_shape_noise_full.yml

```diff
@@ -73,7 +73,7 @@ stages:
     - name: TXTwoPointPlotsFourier 
 
```

Comment:
TXTwoPointLensCat seems to not exists as a stage

Reply:
fixed